frontend/src/components/pages/FridgeManagement/FridgeCheckIns/DeleteShiftModal.tsx

```diff
@@ -0,0 +1,72 @@
+import {
```

Comment:
hmm looking at it, this seems like almost the same code as `DeleteScheduleModal` 
Can we extract that as a `GeneralDeleteShiftModal` and pass in the text for the title, body, and button as props?
That way we don't need to pass in a `dateRange` either.